packages/veraswap-sdk/src/calls/getTransferRemoteWithKernelCalls.ts

```diff
@@ -74,7 +74,7 @@ export async function getTransferRemoteWithKernelCalls(
     };
```

Comment:
TODO: Add calls to add Hyp7579Router as an owner, this way account becomes pre-configured for future crosschain calls.